### PR TITLE
M2-6214: Merge latest from M2-4578-shell-account

### PR DESCRIPTION
**Pull request by mbanting**
### 📝 Description

🔗 [Jira Ticket M2-6214](https://mindlogger.atlassian.net/browse/M2-6214)

Changes include:

- Merges all changes since last merge 7d0c9f0dfe4e46634a3ca84d283ee52e186130d9 from `feature/M2-4578-shell-account` 
- Resolved conflicts
   - src/resources/app-en.json [[#1580](https://github.com/ChildMindInstitute/mindlogger-admin/pull/1580/files#diff-8b07a3883b40e08cd10b693229269699adc9846c6bb21b4f9e58b072c4b39fe6R1097)]
   - src/resources/app-fr.json [[#1580](https://github.com/ChildMindInstitute/mindlogger-admin/pull/1580/files#diff-95c0870b84678947552a3b3e718e4c2ee5e8255384ffebff12a21e7dc87ff944R1096)]
   - src/shared/components/Tabs/DefaultTabs/DefaultTabs.tsx [[#1566](https://github.com/ChildMindInstitute/mindlogger-admin/pull/1566/files#diff-01ec18ab58aa565bf1aef280d8fd21c327338a1e67f4d6274d42a6b19cb27964R82)]
   - src/shared/components/Tabs/Tabs.types.ts [[#1566](https://github.com/ChildMindInstitute/mindlogger-admin/pull/1566/files#diff-6d0c8a74f5a5987f22402993eab0bc3a96c4456d26734bac386ffa43ec989b5fR28), [#1568](https://github.com/ChildMindInstitute/mindlogger-admin/pull/1568/files#diff-6d0c8a74f5a5987f22402993eab0bc3a96c4456d26734bac386ffa43ec989b5fR29),[#1593](https://github.com/ChildMindInstitute/mindlogger-admin/pull/1593/files#diff-6d0c8a74f5a5987f22402993eab0bc3a96c4456d26734bac386ffa43ec989b5fR30-R31) ]

### CODE REVIEW

File: commit_msg_check_flag
Comment (farmerpaul) [inline commit_msg_check_flag:1]: Nit: I think maybe this shouldn't be part of the commit? Perhaps a relic of husky scripts?

**Review by farmerpaul — APPROVED** [review]
Looks good, I just had one nit to point out, but I went through every file (roughly). Seems changes were almost entirely unrelated to our work and I didn't notice anything risky in the merge conflicts. Nice job!

File: commit_msg_check_flag
Comment (mbanting) [reply commit_msg_check_flag:1]: Amazing @farmerpaul . Like finding a needle in a haystack. It's been removed. 

Thanks for the thorough review of a large PR. I'll aim to perform this on a regular basis so the updates are smaller.
